boards/apollo3/lora_things_plus/src/main.rs

```diff
@@ -142,6 +142,18 @@ type BME280Sensor = components::bme280::Bme280ComponentType<
 type TemperatureDriver = components::temperature::TemperatureComponentType<BME280Sensor>;
 type HumidityDriver = components::humidity::HumidityComponentType<BME280Sensor>;
 
+#[cfg(feature = "atecc508a")]
+type Verifier = capsules_extra::atecc508a::Atecc508a<'static>;
+#[cfg(feature = "atecc508a")]
+type SignatureVerifyInMemoryKeys =
+    components::signature_verify_in_memory_keys::SignatureVerifyInMemoryKeysComponentType<
+        Verifier,
+        1,
+        64,
+        32,
+        64,
```

Comment:
Could be good to document these magic numbers.

Reply:
Yeah rustfmt putting those on separate lines really highlights them. They aren't interesting at this layer---they are fixed based on the signature algorithm. They only matter when defining the signature algorithm, where they should be documented.

Reply:
Would it make sense to have the component define a few 

    const SIG_TYPE_AES128 = (64,32,64);

or similar things? Implicitly, this is fixing the type of signatures we support here, no? (I guess it's sort of broader because it's really just fixing to "anything with key length X, hash length Y, etc"; but that's not really how the rest of the stack works at the moment IIUC)
Still, for the moment, it seems like defining constants that name the numbers is maybe a reasonable middle ground?